pilot/pkg/networking/core/v1alpha3/tracing.go

```diff
@@ -419,29 +419,19 @@ func buildServiceTags(metadata *model.NodeMetadata) []*tracing.CustomTag {
 	}
 }
 
-func configureSampling(hcmTracing *hpb.HttpConnectionManager_Tracing, providerPercentage float64, proxyCfg *meshconfig.ProxyConfig) {
+func configureSampling(hcmTracing *hpb.HttpConnectionManager_Tracing, providerPercentage float64) {
```

Comment:
seems the second param passed in is not from provider

Reply:
see proxyConfigSamplingValue(proxyCfg)